frontend/src/constants/constants.ts

```diff
@@ -6,6 +6,7 @@ export const MOONBEAM_WSS = 'wss://wss.api.moonbeam.network';
 export const WALLETCONNECT_ASSETHUB_ID = 'polkadot:68d56f15f85d3136970ec16946040bc1';
 
 export const TRANSFER_WAITING_TIME_SECONDS = 6000;
+export const BRLA_MAXIMUM_LEVEL_2_AMOUNT_UNITS = 1000000; // TODO get the actual value from BRLA.
```

Comment:
Why do we even need this? I created the proper check in a different PR. Do we still need to merge those changes in here?